runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/ComputationState.java

```diff
@@ -126,6 +127,13 @@ public ImmutableList<KeyedGetDataRequest> getKeysToRefresh(
     return activeWorkState.getKeysToRefresh(refreshDeadline, sampler);
   }
 
+  @SuppressWarnings(
```

Comment:
add test to avoid this? or just remove this part for now and add when it actually is used.